lib/reply.js

```diff
@@ -647,6 +654,29 @@ function notFound (reply) {
   }
 }
 
+function serialize (context, data, statusCode) {
```

Comment:
It'd be really helpful if we could start adding jsdoc blocks to functions like this one and `getSchemaSerializer`.

Reply:
I should have done 👍